### PR TITLE
Deploy simple projects to gh-pages using Gulp

### PR DESCRIPTION
**Pull request by pedromartos**

### CODE REVIEW

**Comment by rodrigovdb** [comment]
It's beautiful! I almost cry :'-)

**Comment by viniciusalonso** [comment]
@pedromartos cool article ! And you elaborated a good solution :)
I recommend use javascript syntax highlight, for example:

``` js
var gulp = require('gulp');

    gulp.task('foobar', function(){
      console.log("\n\nHELLO WORLD!!!\n\n");
    });
```

File: 2015-04-10-deploying-simple-html-projects-to-ghpages-using-gulp.markdown [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+    gulp.task('watch', function() {
+      gulp.watch('./src/css/**/*.scss', ['sass']);
+      gulp.watch('./src/js/**/*.coffee', ['scripts']);
+    });
+
+Then we update our `default` task to:
+
+    gulp.task('default', ['sass', 'scripts', 'server', 'watch']);
+
+Now right after the server starts Gulp will watch the files we told him. So for example, when we change our `./src/css/main.css.scss`, Gulp will execute the `sass` task and recompile the file `./build/css/main.css` that our server is watching and will reload it (Actually BrowserSync will inject the changes into the main.css that are already loaded on the browser without even reloading the browser. Told you guys BrowserSync is awesome).
```
Comment (fellipebrito) [inline 2015-04-10-deploying-simple-html-projects-to-ghpages-using-gulp.markdown:161]: I'd add a HUGE MAGIC GIF HERE... this is really the best thing ever in frontend development's world.

**Comment by fellipebrito** [comment]
Dude, it is one of the best tech blog posts I've ever read. It is informative, it has important code, right on point gif animations. It is a long post, but it goes so easy that you barely perceive it is almost 150 lines.

:clap: :clap: :clap: :clap: :clap: :clap:

**Comment by fellipebrito** [comment]
This "JS" line that @viniciusalonso told you. It can be used anywhere, there is syntax highlighting for ruby, bash, html, clojure, etc....

**Pull request merged**